browser/brave_content_browser_client.cc

```diff
@@ -996,6 +988,13 @@ BraveContentBrowserClient::CreateThrottlesForNavigation(
                               Profile::FromBrowserContext(context))))
     throttles.push_back(std::move(reduce_language_navigation_throttle));
 
+  // Debounce
+  if (auto debounce_throttle =
```

Comment:
Hmm, not technically related to this PR, but shouldn't this also only happen on `isMainFrame`?

Reply:
other throttles incorrectly have that logic in BraveContentBrowserClient. DebounceNavigationThrottle handles it internally

Reply:
https://github.com/brave/brave-core/pull/14185/files#diff-1d6ae63a3192724d92fc2dfccbe4dc130ae4309bd355348bb1990027983ab56cR66

Reply:
Okay, because I'm pretty sure we didn't use to handle this correctly. https://github.com/brave/brave-core/pull/14185/files#diff-1d6ae63a3192724d92fc2dfccbe4dc130ae4309bd355348bb1990027983ab56cR119 seems like the right check though.